ansible/roles/skywalking/vars/main.yml

```diff
@@ -15,3 +15,10 @@
 
 ---
 skywalking_version: "9.5.0"
+sw_server_port: "8080"
+sw_oap_server_port: 12800
+
+env_vars:
```

Comment:
Let's prefix this with `sw_ui_`, because I believe we will have `sw_oap_env_vars` soon